From ac05f990db211a27a950d61ef4bf47335ced7cc4 Mon Sep 17 00:00:00 2001
From: Ryan Lortie <desrt@desrt.ca>
Date: Sat, 18 Sep 2010 12:52:58 -0400
Subject: [PATCH] Fix header cleaup fallout when using system pcre

Using the internal pcre has the side effect of exposing gregex.c to
glib.h.  When we use the system one, we lose that, so we need to
explicitly include the things we use (glist, gatomic, etc..)
---
 glib/gregex.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/glib/gregex.c b/glib/gregex.c
index f21fbe8..b62bda7 100644
--- a/glib/gregex.c
+++ b/glib/gregex.c
@@ -32,6 +32,10 @@
 #include "gtypes.h"
 #include "gregex.h"
 #include "glibintl.h"
+#include "glist.h"
+#include "gmessages.h"
+#include "gstrfuncs.h"
+#include "gatomic.h"
 
 /**
  * SECTION:gregex
-- 
1.7.1

